src/environment.js

```diff
@@ -135,6 +135,13 @@ export default class Environment extends EventEmitter {
       this.dir = process.cwd();
     }
 
+    if (!this.dest) {
+      this.dest = 'sassdoc';
+    }
+
+    this.dest = this.resolve(this.dest, this.destCli);
+    this.destName = path.relative(process.cwd(), this.dest);
```

Comment:
You mean `env.display('theme')`? Not possible since the display value for the theme is figured out at the same time as `theme`, depending if it's a package or a path. Or `env.display()` would just be a proxy to the `env.displayTheme` variable.